### ISSUE TITLE
Need to check redirect_to against both http and https

### ISSUE THREAD

**Issue by mitchell360** [miled/wordpress-social-login#27]
wsl.authentication.php line 525 should be enhanced to match both http and https.

```php
if( set_url_scheme(get_option( 'wsl_settings_redirect_url' ), 'http') != site_url() && set_url_scheme(get_option( 'wsl_settings_redirect_url' ), 'https') != site_url()){
		$redirect_to = get_option( 'wsl_settings_redirect_url' );
}

```

**Comment by mitchell360** [comment]
Just curious if this was included or not.

**Comment by miled** [comment]
Honestly I didn't see the point in adding this verification. I also closed this issue as well as #25 because they are now two years old and I'm trying to sort out the repo tracker.

Sorry for not adding an appropriate explanation before closing them.
